microcosm_eventsource/stores/rollup.py

```diff
@@ -130,6 +135,13 @@ def _search_query(self, aggregate=None, **kwargs):
             **kwargs
         )
 
+        if offset is not None:
```

Comment:
You can replace it with `self.container_store._paginate(query, offset=offset, limit=limit)`